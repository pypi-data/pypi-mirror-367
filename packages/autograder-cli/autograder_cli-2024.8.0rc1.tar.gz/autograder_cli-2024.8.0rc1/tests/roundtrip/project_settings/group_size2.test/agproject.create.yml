project:
  name: Test Project e5603a33b7254df7abb2d015ea7766b1
  timezone: America/Chicago
  course:
    name: Test Course
    semester: Summer
    year: 2014
  settings:
    min_group_size: 1
    max_group_size: 2
