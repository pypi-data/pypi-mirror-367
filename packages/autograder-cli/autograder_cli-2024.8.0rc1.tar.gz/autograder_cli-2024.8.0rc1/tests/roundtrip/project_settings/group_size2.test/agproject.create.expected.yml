project:
  name: Test Project e5603a33b7254df7abb2d015ea7766b1
  timezone: America/Chicago
  course:
    name: Test Course
    semester: Summer
    year: 2014
  settings:
    max_group_size: 2
